src/renderers/dom/client/wrappers/__tests__/ReactDOMSelect-test.js

```diff
@@ -537,4 +537,22 @@ describe('ReactDOMSelect', function() {
       "Cannot set property 'pendingUpdate' of null"
     );
   });
+
+  it('should select grandchild options nested inside an optgroup', function() {
+    var stub =
+      <select value="b" onChange={noop}>
+        <optgroup label="group">
+          <option value="a">a</option>
+          <option value="b">b</option>
+          <option value="c">c</option>
+        </optgroup>
+      </select>;
+    var container = document.createElement('div');
+    stub = ReactDOM.render(stub, container);
+    var node = ReactDOM.findDOMNode(stub);
```

Comment:
You don't need to do a findDOMNode here.  The value returned by render is the node, so `var node = ReactDOM.render(stub, container);` is sufficient.